dindex_store/discovery_index.py

```diff
@@ -134,5 +121,7 @@ def find_path(
         return self.__graph_index.find_path(
             source_node_id, target_node_id, max_len)
 
-    def fts_query(self, keywords, search_domain, max_results, exact_search=False) -> List:
-        return self.__fts_index.fts_query(keywords, search_domain, max_results, exact_search)
+    # TODO: search over "search_domain", return top-"max_results",
```

Comment:
you changed the signature, but we need the previous one: def fts_query(self, keywords, search_domain, max_results, exact_search=False) -> List